types/react-hexgrid/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
New packges cannot disable these rules. If the rule need to be disabled on certain line, disable it there so that it gives a chance to verify if thats correct.